src/main/java/com/flightstats/hub/channel/LinkBuilder.java

```diff
@@ -5,10 +5,7 @@
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.flightstats.hub.app.HubProvider;
 import com.flightstats.hub.metrics.ActiveTraces;
-import com.flightstats.hub.model.ChannelConfig;
-import com.flightstats.hub.model.ChannelContentKey;
-import com.flightstats.hub.model.ContentKey;
-import com.flightstats.hub.model.DirectionQuery;
+import com.flightstats.hub.model.*;
```

Comment:
pet peave - can we avoid wildcard imports? I find explicit import helpful in PRs

Reply:
Code I write doesn't use wildcard imports. I'm just trying to restore the master branch with the code that was there before the rollback. That said I'll go make the change.

Reply:
I gotcha.